src/platform/silabs/wifi/rs911x/platform/efx32_ncp_host.c

```diff
@@ -42,35 +40,37 @@
 #include "sl_board_control.h"
 #endif // SL_BOARD_NAME
 
-static bool dma_callback(unsigned int channel, unsigned int sequenceNo, void * userParam);
+#include "sl_si91x_ncp_utility.h"
+#include "spi_multiplex.h"

```

Comment:
Theses files are already included, why do they need to be seperate?

Reply:
They need not be separate, moved them back.